### DISCUSSION TITLE
Allowing ".yarn/releases" and ".yarn/plugins" to be gitignore'd

### DISCUSSION THREAD

**Discussion by cakoose** [yarnpkg/berry#2822] [Ideas]
I'm new to Yarn 2.  I noticed this guidance on what to put in my ".gitignore": https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

> `.yarn/releases` and `.yarn/plugins` contain the Yarn releases used in the current repository (as defined by `yarn set version`). You will want to keep them versioned (this prevents potential issues if, say, two engineers use different Yarn versions with different features).

I see why the repo must contain the release and plugin versions, but I would like to avoid committing the actual bundles.  I think this is similar to how the repo contains "yarn.lock" to track package versions but doesn't need to include the package bundles ("node_modules" and ".yarn/cache").

My current (possibly mistaken) understanding:
* In theory, we don't need to commit the release and plugin bundles to the repo.  We could just commit the version numbers and keep the actual bundles in ".yarn/cache", which could be gitignore'd.
* For that to work, Yarn 2 would need to check the release and plugin versions against the cache and download new bundles if necessary.
* Because the bundles aren't actually _that_ big, Yarn 2 decided not to bother with that extra codepath.

Is that correct?

_If_ that's correct, then I'd just like to mention my support for not including the bundles in the repo.  Wondering if anybody else feels the same way or if it's just me? 🤔

**Comment by arcanis** [comment]
Duplicate of https://github.com/yarnpkg/berry/issues/969
